test/locale_gen_params_test.dart

```diff
@@ -96,6 +96,16 @@ void main() {
                 .outputDir,
             'lib/util/mylocale/');
       });
+      test('Test set asset path error handling', () {
+        try {
+          final result = LocaleGenParams.fromYamlString('locale_gen',
+                  'name: test\n\nlocale_gen:\n  languages: [\'en\',\'fr\']\n  output_path: \'util/mylocale\'')
+              .outputDir;
+          expect('This should not succeed', result);
```

Comment:
Shouldn't we check if this throws an ArgumentError instead of `expect('This should not succeed', result);`?